crates/swc/tests/simple.rs

```diff
@@ -36,6 +36,36 @@ fn compile(src: &str, options: Options) -> String {
         .unwrap()
 }
 
+#[test]
+fn help() {
```

Comment:
Did you add this intentionally?

Reply:
nope sorry that's the test I use for trying out issues locally 😄 rebased